docs/source/trainable_reconstruction.rst

```diff
@@ -0,0 +1,33 @@
+.. automodule:: lensless.trainable_recon
+
+
+   Trainable reconstruction API
+   ------------------
```

Comment:
don't need extend "---" for the whole length of heading?

Reply:
I doesn't seem to cause any problem, but I will fix it.